docs/banzai/index.rst

```diff
@@ -219,6 +312,6 @@ catalog will be used.
 
 
 Reference/API
-=============
+-------------
```

Comment:
Why the switch of heading style?

Reply:
I thought this makes more sense as its own section, rather than as a subsection.